db/migrate/20160505005253_add_send_count_to_invitations.rb

```diff
@@ -0,0 +1,5 @@
+class AddSendCountToInvitations < ActiveRecord::Migration
+  def change
+    add_column :invitations, :send_count, :integer, null: true, default: 0
```

Comment:
I'm pretty sure we want an index on this column; maybe a partial index as described in this article:
http://rny.io/rails/postgresql/2013/08/20/postgresql-indexing-in-rails.html